### ISSUE TITLE
Issue with k8s.io/docs/tutorials/kubernetes-basics/deploy-interactive/

### ISSUE THREAD

**Issue by abhi-vaidya** [kubernetes/website#7237]
kubectl proxy command hungs at starting server on port 8001

**This is a...** 

- [ ] Feature Request
- [ ] Bug Report

**Problem:**

**Proposed Solution:**

**Page to Update:**
http://kubernetes.io/...

**Comment by errordeveloper** [comment]
I think can see that you get an impression that the command still runs in the foreground, but actually, if you were to ignore that and just click on the next step, you will see that it all works okay.

I'm tempted to close this, ping @BenHall.

**Comment by BenHall** [comment]
Indeed. Sadly `kubectl proxy` doesn't output any messages so it looks like nothing is happening, but it's actually by design. 

We've tried to make this clear in the docs, but we'll have another look.

**Comment by errordeveloper** [comment]
Maybe make the command `kubectl proxy &`, so it's clearer?
